grading/python/submitty_router.py

```diff
@@ -111,7 +111,7 @@ def write_sequence_file(self, obj, status, message_type):
         outfile.write(f'{start}: {message}\n')
       else:
         str_lines = [str(x) for x in message_lines ]
-        outfile.write(f'{start}: {"<br>".join(str_lines)}\n')
+        outfile.write(f'{start}: {"<br/>".join(str_lines)}\n')
```

Comment:
This change shouldn't be necessary since `<br>` tags don't need to be closed.

Reply:
Oops, forgot the suggestion (and apparently can't edit in afterwards)
```suggestion
        outfile.write(f'{start}: {"<br>".join(str_lines)}\n')
```